cmd/insprd/operators/kafka/nodes/converter.go

```diff
@@ -98,6 +98,7 @@ func (no *NodeOperator) withBoundary(app *meta.App) k8s.ContainerOption {
 			parent, chName, _ := metautils.RemoveLastPartInScope(resolved)
```

Comment:
Suggestion: Make this function a little more legible: separate some lines

Reply:
I didn't implement this actually. This can be changed in CORE-416 tho